packages/a/aws-sdk-cpp/xmake.lua

```diff
@@ -18,7 +18,7 @@ package("aws-sdk-cpp")
 
     on_load(function (package)
         if package:config("http_client") then
-            package:add("deps", "libcurl")
+            package:add("deps", "libcurl", {configs = {openssl = false}})
```

Comment:
why?

Reply:
[CMakeLists.txt](https://github.com/aws/aws-sdk-cpp/blob/ecf0320fefbd57ef1744c2b5c539e4466cd252ed/CMakeLists.txt#L195)
> libcurl and aws-sdk-cpp-core may link to different libcrypto, which leads to some issues for shared build.

I would like to rule out if this causes compilation errors😅

Reply:
By default, they should all be using xmake-repo or openssl on the system, unless it uses the system libcurl library.

Also disabling libcurl's openssl may cause http to lack ssl features.